src/writer/proposal.ts

```diff
@@ -233,8 +233,10 @@ export async function action(body, ipfs, receipt, id): Promise<void> {
     scores_total: 0,
     scores_updated: 0,
     votes: 0,
-    validation
+    validation,
+    flagged: containsFlaggedLinks(msg.payload.body) ? 1 : 0
```

Comment:
It should be the same as `update-proposal.ts` where it is `flagged: +containsFlaggedLinks(msg.proposal.body)` or we need to update `update-proposal.ts` like here 😅